### ISSUE TITLE
More import options

### ISSUE THREAD

**Issue by asg017** [asg017/dataflow#10]
Only observablehq and local imports are supported now. Here are other places we should support importing from:

- [ ] Any `.js` file on the internet that has a `export default function define()` definition
- [ ] Any local `.js` file that has a `export default function define()` definition
- [ ] Github (proxy to raw.githubusercontent.com)
- [ ] NPM (maybe should just document how to use https://www.jsdelivr.com instead of doing npm resolution ourselves

**Comment by Jiahao01121** [comment]
+1 on the npm import. Is there a way to use jsdelivr now?

**Comment by asg017** [comment]
@Jiahao01121 yup, Dataflow uses the same [`require`](https://github.com/observablehq/stdlib#require) functions as the notebooks on Observable, which uses [d3-require](https://github.com/d3/d3-require) under the hood. So

```javascript
// notebook.ojs
moment = require("moment")

moment(new Date)
```

imports moment from jsdelivr.

**Comment by declann** [comment]
+1 for local imports, this can be a big + for dataflow vs observablehq for local js dev

**Comment by a10k** [comment]
For gists, etc., this works for me in the import resolver:
~~~js
fetch(path)
      .then((res) => res.text())
      .then((js) => {
        const blob = new Blob([js], {
          type: 'text/javascript',
        });
        const objectURL = URL.createObjectURL(blob);
        return objectURL;
      })
~~~
